front/src/app/api/logs/[id]/connectTag/route.ts

```diff
@@ -0,0 +1,16 @@
+import { LogsData } from "@/services/LogsData";
+import { NextResponse } from "next/server";
+
+export async function POST(
+    req: Request,
+    { params }: { params: { id: string } },
```

Comment:
Might be nice for readability for this to be typed outside e.g ```type QueryParams = { id : string }``` and then you could us it like ` { params : QueryParams}`

Reply:
but maybe thats just my preference :P